lib/epochs.js

```diff
@@ -151,6 +151,19 @@ module.exports = function Epochs (ssb) {
 
         cb(null, result)
       })
+    },
+
+    tieBreak (epochs) {
+      if (!epochs || !Array.isArray(epochs))
+        throw Error('tieBreak requires an Array of epochs')
+
+      const keys = epochs
+        .map(epoch => epoch.epochKey.toString('base64'))
+        .sort()
```

Comment:
https://github.com/ssbc/ssb-group-exclusion-spec#43-tie-breaking-rule


> The tie-breaking rule is: among the forked epochs, sort their epoch keys by lexicographic order (in hexadecimal encoding) and the epoch with the smallest epoch key is considered the winner.


Needs to be hex encoding not base64!